lib/ts/recipe/emailpassword/types.ts

```diff
@@ -325,6 +325,40 @@ export type APIInterface = {
                 }
               | GeneralErrorResponse
           >);
+
+    passwordStateGET?: (input: {
+        email: string;
+        session: SessionContainerInterface;
+        options: APIOptions;
+        userContext: UserContext;
+    }) => Promise<
+        | {
+              status: "OK";
+              exists: boolean;
+          }
+        | GeneralErrorResponse
+    >;
+
+    updatePasswordPOST?: (input: {
+        details: {
+            email: string;
+            newPassword: string;
+            oldPassword?: string;
+        };
+        session: SessionContainerInterface;
```

Comment:
session verification here should not have `overrideGlobalClaimValidators: () => []` in our api implementation